Makefile

```diff
@@ -62,13 +62,13 @@ else
 	go build $(BUILD_FLAGS) -o build/gaiakeyutil ./cmd/gaia/cmd/gaiakeyutil
 endif
 
-build-linux:
+build-linux: vendor-deps
```

Comment:
what is the reasoning behind this change?

I would not like to wait vendor-deps every time I want to build a binary.

Reply:
Yeh, I know, but I still prefer to use separate targets like

```
all: vendor-deps build-linux
```

instead of adding dependencies

Reply:
Why? You can do both

Reply:
```
$ time make get_vendor_deps
--> Running dep

real    0m12.096s
user    0m0.464s
sys     0m4.560s
```

that's why

Reply:
If you run it again, nothing will happen. `install` now depends on the `vendor-deps` file to generate `vendor` hash info

Reply:
```
$ time make get_vendor_deps
--> Running dep

real    0m8.603s
user    0m0.320s
sys     0m3.900s
```

Reply:
Does not matter. My point is it's slow (even if it's just verifying, not downloading, libs).

Reply:
By how much does it slow build down? Please reproduce the following, timings are negligible:

```
alessio@bangalter:~/.../cosmos/cosmos-sdk$ time make vendor-deps
make: 'vendor-deps' is up to date.

real	0m0.014s
user	0m0.013s
sys	0m0.001s
```

Reply:
I think this change looks good here!

Reply:
Sorry. Should've tried it with cosmos sdk (thought it will be ~ same timings).